en/theme/material/assets/css/theme.css

```
@@ -15,336 +15,396 @@
  */
- 
-@import url('partials/_typography.css');
-@import url('partials/_button.css');
-@import url('partials/_icon.css');
-@import url('partials/_search.css');
-@import url('partials/_card.css');
-@import url('partials/_footer.css');
-@import url('partials/_sidebar.css');
-@import url('partials/_header.css');
+
+@import url("partials/_typography.css");
```

Stylelint (17.14.0)

[error] Expected "url("partials/_typography.css")" to be ""partials/_typography.css"" (import-notation)

(import-notation)